src/firmware/boards/legacy_dongle_stm32f4/normal.c

```diff
@@ -447,27 +452,30 @@ static void send_drive_packet(const void *packet, const uint8_t *serials)
     // Message purpose
     mrf_write_long(address++, 0X0FU);
 
-    // Write out the payload sent from the host, interleaved with the prefix
-    // byte for each robot.
+    // Write out the payload sent from the host
     const uint8_t *rptr = packet;
```

Comment:
Should this not be cast to a (uint8_t) before assignment?

Reply:
tecnically `(uint8_t*)`, but yeah. Changed